seerep-srv/seerep-server/src/fb-meta-operations.cpp

```diff
@@ -144,4 +144,130 @@ grpc::Status FbMetaOperations::DeleteProject(grpc::ServerContext* context,
   return grpc::Status::OK;
 }
 
+grpc::Status
+FbMetaOperations::GetOverallTimeInterval(grpc::ServerContext* context,
+                                         const flatbuffers::grpc::Message<seerep::fb::UuidDatatypePair>* request,
+                                         flatbuffers::grpc::Message<seerep::fb::TimeInterval>* response)
+{
+  (void)context;  // ignore that variable without causing warnings
+  auto requestRoot = request->GetRoot();
+
+  std::string uuid = request->GetRoot()->projectuuid()->str();
+  boost::uuids::string_generator gen;
+  auto uuidFromString = gen(uuid);
+
+  std::vector<seerep_core_msgs::Datatype> dt_vector;
+
+  seerep::fb::Datatype casted_datatype = static_cast<seerep::fb::Datatype>(requestRoot->datatypes());
+  if (casted_datatype == seerep::fb::Datatype_Image)
+  {
+    dt_vector.push_back(seerep_core_msgs::Datatype::Image);
+  }
+  else if (casted_datatype == seerep::fb::Datatype_PointCloud)
+  {
+    dt_vector.push_back(seerep_core_msgs::Datatype::PointCloud);
+  }
+  else if (casted_datatype == seerep::fb::Datatype_Point)
+  {
+    dt_vector.push_back(seerep_core_msgs::Datatype::Point);
+  }
+  else
+  {
+    dt_vector.push_back(seerep_core_msgs::Datatype::Unknown);
+  }
+
+  seerep_core_msgs::AabbTime timeinterval = seerepCore->getOverallTimeInterval(uuidFromString, dt_vector);
```

Comment:
All packages in the seerep-server package should catch all errors. Take a look at the other functions and add the try-catch here